editor/svgcanvas.js

```diff
@@ -4596,7 +4596,7 @@ this.setSvgString = function(xmlString) {
 		svgedit.path.clearData();
 		svgroot.appendChild(selectorManager.selectorParentGroup);
 		
-		addCommandToHistory(batchCmd);
+		if(!preventUndo) addCommandToHistory(batchCmd);
```

Comment:
nit: add a space between the if and the (

also, can you please correct the other two instances of that in this file?

Reply:
I only found one other instance of it in this branch, but i've changed that now.

I might have done it in that style in other commits / pull requests. Sorry about that. I didn't realize that was the coding style of this project. I'll try to do that in the future.